.github/workflows/deploy.yml

```diff
@@ -0,0 +1,81 @@
+# SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+name: deploy
+
+on:
+  pull_request:
+
+jobs:
+  deploy:
+    if: ! github.event.pull_request.draft
+    runs-on: ubuntu-latest
+    name: Deploys to netlify
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+
+      - name: Get deployment id for usage in the URL
+        id: deploy-url
+        run: echo "::set-output name=url::${{ github.event.number }}--${{ github.head_ref }}"
+
+      - name: Cache yarn cache
+        uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-16-yarn-${{ hashFiles('**/yarn.lock') }}
```

Comment:
should this number be hardcoded?

Reply:
We don't use multiple node versions here. So it can be removed.

Reply:
Well, it's the same as with the lint.yml workflow.